examples/hetero/load_csv.py

```diff
@@ -16,7 +16,7 @@
 
 def load_node_csv(path, index_col, encoders=None, **kwargs):
     df = pd.read_csv(path, index_col=index_col, **kwargs)
-    mapping = {index: i for i, index in enumerate(df.index.unique())}
```

Comment:
I am not sure I understand the change. `pd.read_csv(index_col=index_col)` will move the `index_col` to `df.index`, so we should not be able to access it via `df[index_col]`.

Reply:
You are right, I loaded the dataframes outside of the function and didnt reassign the indexes.